src/components/Navbar/NavItem.js

```diff
@@ -0,0 +1,32 @@
+// Dependencies
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+// Components, actions, assets, etc..
+import Button from 'components/Button/Button';
+
+const NavItem = (props) => {
+  const { name, linkTo, children, navState, setNavState } = props;
+  const isActive = navState === name ? 'navbar__link--active' : null;
+
+  return name === 'Donate' ? (
+    <Button type="outlined" link="donate">
```

Comment:
@mcheng09 i think marco needs to do a `/` yea?
```suggestion
    <Button type="outlined" link="/donate">
```